module-apps/application-call/windows/CallWindow.hpp

```diff
@@ -29,6 +29,14 @@ namespace gui
 
         std::unique_ptr<app::call::CallWindowContract::Presenter> presenter;
 
+        std::set<audio::EventType> devices_connected{};
+        bool isLouderSpeakerON = false;
```

Comment:
```suggestion
        bool isLoudspeakerOn = false;
```

Why not just like that?